recipes/r-autoplotly/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = '0.1.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-autoplotly
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: autoplotly_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/autoplotly_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/autoplotly/autoplotly_{{ version }}.tar.gz
+  sha256: 40d25f0fcec009260d504d4e55a299347ce8584c3bf8838d81e0a6d68039a1de
+
+build:
+  number: 0
+  skip: true  # [win32]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-ggfortify
+    - r-ggplot2
+    - r-plotly
+
+  run:
+    - r-base
+    - r-ggfortify
+    - r-ggplot2
+    - r-plotly
+
+test:
+  commands:
+    - $R -e "library('autoplotly')"  # [not win]
+    - "\"%R%\" -e \"library('autoplotly')\""  # [win]
+
+about:
+  home: https://cran.r-project.org/web/packages/autoplotly/index.html
+  license: Apache 2.0
+  summary: Functionalities to automatically generate interactive visualizations for popular statistical
+    results supported by 'ggfortify', such as time series, PCA, clustering and survival
+    analysis, with 'plotly.js' <https
```

Comment:
Could you please manually fix the summary? This is the same bug that truncates the home URL. One of these days conda-forge will migrate to conda-build 3 and we won't have to deal with this bug anymore.